packages/devkit/src/generators/generate-files.ts

```diff
@@ -27,12 +36,14 @@ const { logger } = requireNx();
  * @param srcFolder - the source folder of files (absolute path)
  * @param target - the target folder (relative to the tree root)
  * @param substitutions - an object of key-value pairs
+ * @param overwriteStrategy - behaviour when the target file already exists. Defaults to Overwrite
  */
 export function generateFiles(
   tree: Tree,
   srcFolder: string,
   target: string,
-  substitutions: { [k: string]: any }
+  substitutions: { [k: string]: any },
+  overwriteStrategy: OverwriteStrategy = OverwriteStrategy.Overwrite
```

Comment:
Please add this flag within an `options` object. This will make future flags not a breaking change.